packages/Popover/src/helpers/getPosition.js

```diff
@@ -61,21 +62,31 @@ const getAnchor = (rect, align) => {
   return positions[align]();
 };
 
-export const getContentCoordinates = ({ rect, targetRect, scrollRect, align, offset = 12 }) => {
+export const getContentCoordinates = ({ rect, targetRect, scrollRect, align, offset = 12, edge }) => {
   const anchor = getAnchor(targetRect, align);
 
+  const getContentXByEdge = edge => {
```

Comment:
Is there a reason to pass `edge` as a parameter to this function, instead of using is directly (as we do for `rect` or `targetRect`, for instance)?

Reply:
Oh no reason, I forgot to remove it when I move around the functions 😿